server/src/main/kotlin/org/javacs/kt/signaturehelp/SignatureHelp.kt

```diff
@@ -97,8 +101,13 @@ private fun candidates(call: KtCallExpression, file: CompiledFile): List<Callabl
     return emptyList()
 }
 
-private fun activeDeclaration(call: KtCallExpression, candidates: List<CallableDescriptor>): Int {
-    return candidates.indexOfFirst { isCompatibleWith(call, it) }
+private fun activeSignature(call: KtCallExpression, candidates: List<CallableDescriptor>): Int? {
+    val activeIndex = candidates.indexOfFirst { isCompatibleWith(call, it) }
```

Comment:
Was just about to comment that we could use `indexOfFirstOrNull`, then I found that no such method actually exists in the Kotlin standard library 😄